app/src/app/[lng]/onboarding/setup/page.tsx

```diff
@@ -495,7 +497,7 @@ export default function OnboardingSetup({
             />
           )}
         </div>
-        <div className="bg-white w-full fixed bottom-0 left-0 border-t-4 border-brand flex flex-row py-8 px-8 drop-shadow-2xl hover:drop-shadow-4xl transition-all">
+        <div className="bg-white w-full fixed z-[9999] bottom-0 left-0 border-t-4 border-brand flex flex-row py-8 px-8 drop-shadow-2xl hover:drop-shadow-4xl transition-all">
```

Comment:
Not sure if the z-index needs to be this high but good fix for now. I think the map is around z-index 500 or so, but it has multiple layers.